src/CallbackLater.php

```diff
@@ -21,7 +21,7 @@ class CallbackLater extends Callback
      *
      * @return mixed|null
      */
-    public function set($callback, $args = [])
+    public function set($callback = null, $args = null)
```

Comment:
$fx

Reply:
>     * what is the breaking change?

![image](https://user-images.githubusercontent.com/2228672/89417803-7a173f00-d72f-11ea-9488-2462a402fd36.png)


>     * what is the benefit?

strict item tree type validation and later abstract nesting for Callback

Reply:
`$fx` renamed